src/mono/nuget/Microsoft.NET.Workload.Mono.Toolchain.Manifest/WorkloadManifest.json.in

```diff
@@ -166,6 +166,7 @@
       "version": "${PackageVersion}",
       "alias-to": {
         "win-x64": "Microsoft.NETCore.App.Runtime.AOT.win-x64.Cross.android-x86",
+        "win-arm64": "Microsoft.NETCore.App.Runtime.AOT.win-x64.Cross.android-x86",
```

Comment:
@Redth when you're testing win-arm64, can you validate if this `.exe` runs:

```
> "C:\Program Files\dotnet\packs\Microsoft.NETCore.App.Runtime.AOT.win-x64.Cross.android-x86\6.0.2-mauipre.1.22074.5\tools\mono-aot-cross.exe"
````

If it prints help output, I think that means it would work.